<!--This is an example configuration file from our use at Max Planck Institute for Intelligent Systems. We have a da Vinci stereo endoscope and want to push the images to `/endoscope/left/image_raw` and `/endoscope/right/image_raw`. We use a DeckLink Quad 2.-->


<launch>
    <group>
    <push_ros_namespace namespace="endoscope/left"/>
        <node pkg="hi_decklink_ros" exec="publisher" name="camera" required="true" output="screen">
            <param name="decklink_device" value="DeckLink Quad (4)" />
            <param name="camera_name" value="ecml" />
            <param name="camera_frame" value="endoscope/left_camera" />
            <param name="camera_info_url" value="file://${ROS_HOME}/camera_info/davinci-endoscope-left-stereo.yaml" />
        </node>
    </group>
    <group>
    <push_ros_namespace namespace="endoscope/right"/>
        <node pkg="hi_decklink_ros" exec="publisher" name="camera" required="true" output="screen">
            <param name="decklink_device" value="DeckLink Quad (3)" />
            <param name="camera_name" value="ecmr" />
            <param name="camera_frame" value="endoscope/right_camera" />
            <param name="camera_info_url" value="file://${ROS_HOME}/camera_info/davinci-endoscope-right-stereo.yaml" />
        </node>
    </group>
</launch>

